styles.css

```diff
@@ -351,6 +364,394 @@ a.see-project:active,
   list-style: none;
 }
 
+#footer a {
+  display: flex;
+}
+
 #footer ul li + li {
   margin-left: 1.81rem;
 }
+
+@media only screen and (min-width: 768px) {
+  nav {
+    height: 100px;
+    box-shadow: 0 0.5px 0 #dddfe6;
+    background-image: none;
+    justify-content: center;
+  }
+
+  nav a.menu {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    list-style: none;
+    margin: auto;
+    font-size: 16px;
+    justify-content: space-between;
+    width: 369px;
+  }
+
+  nav ul a {
+    text-decoration: none;
+    color: #1a2236;
+  }
+
+  nav ul a:hover {
+    text-decoration: underline;
+  }
+
+  #headline {
+    background-image: url("./images/image-geometry_1.png"), url(./images/image_dots.png);
+    background-repeat: no-repeat, no-repeat;
+    background-position: right top, right 55.75% bottom;
+    background-size: 48.75% auto, 12.15% auto;
+    padding: 192px 55.3% 13.2% 11.1%;
+    max-width: 1440px;
+    margin: 0 auto 14.6% auto;
+  }
+
```

Comment:
Please make sure your design is the same as the given Figma design. Currently, the position of your background image is a little bit different. 
Your design
![image](https://github.com/Zilola-Nazarova/zilola-portfolio/assets/101523582/cee58620-3397-4b32-94bb-49384569813a)

Figma design
![image](https://github.com/Zilola-Nazarova/zilola-portfolio/assets/101523582/19d76b20-f557-4bf2-9964-dbc31b9eb2b3)